prompt-management/features/advanced/link-to-traces.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: "Link to Traces"
+description: "Connect prompts to execution traces for performance monitoring and analysis"
+---
+
+Linking prompts to traces enables tracking of metrics and evaluations per prompt version. It's the foundation of improving prompt quality over time.
+
+After linking prompts and traces, you will see information about the prompt in the trace's metadata.
+
+For more information about traces and spans, see the [Concepts](/concepts) guide.
+
+## How to Link Prompts to Traces
+
+When you use `getPrompt()` within a trace context, LangWatch automatically links the prompt to the trace:
+
+<Tabs>
+<Tab title="Python SDK">
+
+```python
+import langwatch
+from openai import OpenAI
+
+# Initialize LangWatch
+langwatch.setup()
+
+@langwatch.trace()
+def customer_support_generation():
+    # Get prompt (automatically linked to trace when API key is present)
+    prompt = langwatch.prompt.get_prompt("customer-support-bot")
+
+    # Format messages with variables
+    messages = prompt.format_messages(
+        user_name="John Doe",
+        user_email="john.doe@example.com",
+        input="I need help with my account"
+    )
+
+    # Use with OpenAI client
+    client = OpenAI()
+    completion = client.chat.completions.create(
+        model=prompt.model.split("openai/")[1],
+        messages=messages
+    )
+
+    return completion.choices[0].message.content
+
+# Call the function
+result = customer_support_generation()
+```
+
+</Tab>
+<Tab title="TypeScript SDK">
+
+```typescript
+import { getPrompt } from "langwatch";
+import OpenAI from "openai";
+
+// Initialize LangWatch (tracing is automatic when API key is set)
+const openai = new OpenAI();
+
+async function customerSupportGeneration() {
+  // Get prompt (automatically linked to trace when API key is present)
+  const prompt = await getPrompt("customer-support-bot");
```

Comment:
You can also compile by passing in variables second, not sure if you want to highlight